### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by 2br007**
### Story:
Create Cli able to get argument and print results from DB

### Description:
- Add main logic
- Add basic unit tests

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	}
+	return true
+}
+
+// Get info about provided city.
+func getCityInfo(c string, db *sqlx.DB) CityInfo {
+	query := fmt.Sprintf(
+		`SELECT city_name, temperature, humidity
+		 FROM "weather"
+		 WHERE city_name = '%s'`, c)
```
Comment (0xff00ff) [inline main.go]: i see the sql injection here

File: main.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+}
+
+// Get info about provided number of cities,
+// for example: 10 will print info about 10 cities.
+func getRowsLimitedBy(n string, db *sqlx.DB) []CityInfo {
+	query := fmt.Sprintf(
+		`SELECT city_name, temperature, humidity
+		 FROM "weather"
+		 LIMIT '%s'`, n)
```
Comment (0xff00ff) [inline main.go]: the same with injection

File: main.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+	)
+)
+
+type CityInfo struct {
+	CityName    string `db:"city_name"`
+	Temperature string `db:"temperature"`
+	Humidity    string `db:"humidity"`
+}
+
+var cI CityInfo
```
Comment (0xff00ff) [inline main.go]: image

Comment (0xff00ff) [reply main.go]: the problem is not in the name )

Comment (2br007) [reply main.go]: but still name was ugly)

**Pull request merged**